app/scripts/controllers/specialErrorDetail.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name hmdaPilotApp.controller:SpecialErrorDetailCtrl
+ * @description
+ * # SpecialErrorDetailCtrl
+ * Controller of the hmdaPilotApp
+ */
+module.exports = /*@ngInject*/ function ($scope, $routeParams, $location, $http, HMDAEngine, Session) {
+
+    // Get the list of errors from the HMDAEngine
+    var editType = 'special',
+        editId = $routeParams.EditId,
+        editErrors = HMDAEngine.getErrors();
+
+    $scope.editType = editType;
+    $scope.editId = editId;
+    $scope.siblingEdits = [];
+
+    if (editErrors[editType] && editErrors[editType][editId]) {
+        $scope.editError = editErrors[editType][editId];
+        $scope.siblingEdits = Object.keys(editErrors[editType]).sort();
+        $scope.selectedEditId = editId;
+    } else {
+        $scope.editError = {};
+    }
+
+    if (editId === 'Q595') {
+        if (Session.isVerified(editId)) {
+            $scope.checkboxes = Session.getVerifiedSpecialEdit(editId);
+        } else {
+            $scope.checkboxes = [];
+            for (var i = 1; i <= $scope.editError.errors.length; i++) {
+                $scope.checkboxes[i] = false;
+            }
+        }
+    } else if (editId === 'Q029') {
+        $scope.selectedAnswer = $scope.selectedAnswer || '0';
+        if (Session.isVerified(editId)) {
+            $scope.selects = Session.getVerifiedSpecialEdit(editId);
+        } else {
+            $scope.selects = [];
+            for (var j = 1; j <= $scope.editError.errors.length; j++) {
+                $scope.selects[j] = '0';
+            }
+        }
+    }
+
+    $scope.$watch(function() {
+        return $scope.isLastPage();
+    }, function(isLastPage) {
+        if (isLastPage) {
+            $scope.canVerify = true;
+        }
+    });
+
+    $scope.pageSize = $scope.pageSize || 10;
+    $scope.currentPage = $scope.currentPage || 1;
+
+    $scope.start = function() {
```

Comment:
I was just following your lead from the IRS report ;)

But yes, agreed on a refactor, just didn't want to hold the rest of this back for it.